src/lwc/psElevateTokenHandler/psElevateTokenHandler.js

```diff
@@ -238,19 +234,20 @@ class psElevateTokenHandler {
             };
         });
 
-        iframe.contentWindow.postMessage(
-            {
-                action: MOUNT_IFRAME_EVENT_ACTION,
-                paymentMethod: paymentMethod
-            },
-            this._visualforceOrigin
-        );
+        const message = {
+            action: MOUNT_IFRAME_EVENT_ACTION,
+            paymentMethod: paymentMethod
+        };
+        this.sendIframeMessage(iframe, message, this._visualforceOrigin);
 
         return mountPromise;
     }
-}
 
-/**
- * A new instance of the Elevate tokenization handler
- */
-export default new psElevateTokenHandler();
+
+    sendIframeMessage(iframe, message, targetOrigin) {
```

Comment:
New method extracted out from psElevateTokenHandler.js, this method is overridden at the bottom of the mocked version of this class. [See here](https://github.com/SalesforceFoundation/NPSP/pull/6354/files#r606489039)